tests/integration/tier1/convert-offline-systems/main.fmf

```diff
@@ -1,4 +1,6 @@
-summary: Convert systems that have no access to the Internet
+summary: |
+    Convert systems that have no access to the Internet. This test require having the repositories of original
+    distro available in the satellite server. As of now we only have CentOS7 repositories available.
```

Comment:
Summary should be shorter, 50 characters if possible.
Description can be longer.
But we can fix this whenever.

Reply:
I created a Jira issue for this so we can tackle all the missing stuff in future fix and not block current PR.
Link to the Jira: https://issues.redhat.com/browse/RHELC-824